src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -163,6 +167,7 @@ trait FastSync {
     private var bestBlockHeaderNumber: BigInt = initialSyncState.bestBlockHeaderNumber
 
     private var assignedHandlers: Map[ActorRef, Peer] = Map.empty
+    private var peerRequestsTime: Map[Peer, Date] = Map.empty
```

Comment:
Perhaps `java.time.Instant` or simply `Long` would be more appropriate than `Date`?